src/app/routes/pages/calendar/calendar.component.ts

```diff
@@ -103,7 +93,7 @@ export class CalendarComponent implements OnInit, DoCheck {
     // },
```

Comment:
What is this commented out curly brace for?

Reply:
honestly, i have no clue, wasn't mine

Reply:
this isn't fully ready to be merged but I do want to finish it tomorrow at work

Reply:
Okay got it.  Thanks

Reply:
maybe just delete it so long as it doesn't affect the functionality of the code